packages/header-bar/src/menus/ProfileMenu.js

```diff
@@ -19,6 +19,7 @@ const ProfileMenu = addD2Context((props, { d2 }) => {
     }
 
     const initials = `${currentUser.firstName.charAt(0)}${currentUser.surname.charAt(0)}`;
+    /* eslint-disable */
```

Comment:
Why does eslint need to be disabled here?

Reply:
Also curious about which rule this violates?

On a related note: it is better to disable on a per-rule basis, that way other developers gets a hint about what rule it violates and could potentially fix it.

Reply:
Upon more investigation the code was pointlessly setting the `key` attribute with a non-existing variable "name". The `key` attribute has now been removed (can you just confirm that the attribute was pointless?)

Reply:
Yes, looks pointless to me as well.  Maybe those used to be rendered as an array but as written now it's unnecessary to use a `key` attribute, especially one that's `undefined` 👍

Reply:
Yes I know, it was pure laziness that I didn't specify the rule:  no-undef

Reply:
the "name" variable was not defined anywhere.